components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -0,0 +1,122 @@
++++
+title = "Data Feed"
```

Comment:
should this be Data Feeds?

Reply:
Thanks @susanev - fixed plural needs throughout doc!